src/main/java/domain/card/Card.java

```diff
@@ -0,0 +1,29 @@
+package domain.card;
+
+public class Card {
+
+    private final Symbol symbol;
+    private final Type type;
+
+    public Card(Symbol symbol, Type type) {
+        this.symbol = symbol;
+        this.type = type;
+    }
+
+    public boolean isAceType() {
+        return type.equals(Type.ACE);
```

Comment:
```suggestion
        return type == Type.ACE;
```
Enum의 비교는 **equlas**가 아닌  ``==`` 로 해요 😉
아래의 ``isTenNumberType`` 또한 변경해주세요 🙏

Reply:
피드백 감사합니다 :)
Enum은 선언된 것들만 존재하기 때문에, 유일하다고 볼 수 있기 때문에 `==`을 통해 `동일성` 비교를 하는게 맞을까요?